dist/debian/debian_files_gen.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import string
+import os
+import shutil
+import re
+from pathlib import Path
+
+class DebianFilesTemplate(string.Template):
+    delimiter = '%'
+
+scriptdir = os.path.dirname(__file__)
+
+with open(os.path.join(scriptdir, 'changelog.template')) as f:
+    changelog_template = f.read()
+
+with open(os.path.join(scriptdir, 'control.template')) as f:
+    control_template = f.read()
+
+with open('build/SCYLLA-PRODUCT-FILE') as f:
+    product = f.read().strip()
+
+with open('build/SCYLLA-VERSION-FILE') as f:
+    version = f.read().strip().replace('.rc', '~rc').replace('_', '-')
+
+with open('build/SCYLLA-RELEASE-FILE') as f:
+    release = f.read().strip()
+
+if os.path.exists('build/debian/debian'):
+    shutil.rmtree('build/debian/debian')
+shutil.copytree('dist/debian/debian', 'build/debian/debian')
+
+if product != 'scylla':
+    for p in Path('build/debian/debian').glob('scylla-*'):
+        # pat1: scylla-server.service
+        #    -> scylla-enterprise-server.scylla-server.service
+        # pat2: scylla-server.scylla-fstrim.service
+        #    -> scylla-enterprise-server.scylla-fstrim.service
+        # pat3: scylla-conf.install
+        #    -> scylla-enterprise-conf.install
```

Comment:
Do we need it?

Reply:
this file was copied as is from scylla-java-tools, I didn't change a thing here.

Reply:
This isn't a good response. If it's wrong, let's not copy it. If it's right then ok.

Reply:
We need to keep this just for scylla-cqlsh.install.
And we actually drop scylla-cqlsh.install, since our .deb build script is using .install file for producing mutiple .deb package (sub-package) on single repository.
We install all files to $(CURDIR)/debian/tmp and then picking files by *.install file for each sub-packages.
Since scylla-cqlsh is single package, we just need to install $(CURDIR)/debian/scylla-cqlsh and stop using .install file.

The fix will be like this:
https://github.com/syuu1228/scylla-cqlsh/commit/eeeb1fae24cb9ef63c6c531a89bedf57c2744226

Reply:
took the suggested patch